upup/pkg/fi/cloudup/vsphere/vsphere_cloud.go

```diff
@@ -60,6 +65,8 @@ func NewVSphereCloud(spec *kops.ClusterSpec) (*VSphereCloud, error) {
 	cluster := *spec.CloudConfig.VSphereResourcePool
 	glog.V(2).Infof("Creating vSphere Cloud with server(%s), datacenter(%s), cluster(%s)", server, datacenter, cluster)
 
+	dns_server := *spec.CloudConfig.VSphereCoreDNSServer
+	dns_zone := spec.DNSZone
```

Comment:
This is populated here: https://github.com/vmware/kops/pull/2/files#diff-0b16cb0df6c9387b2c161f3aeb2e032bR510